index.js

```diff
@@ -83,16 +83,22 @@ const queryDns = (version, options) => {
 
 	const socket = dns({
 		retries: 0,
+		maxQueries: 1,
 		socket: dgram.createSocket(version === 'v6' ? 'udp6' : 'udp4'),
 		timeout: options.timeout
 	});
+	socket.retries = 0;
```

Comment:
Can you open an issue on `dns-socket` about this bug and add the bug URL as a code comment here?

Reply:
Yes, I just did an pull request instead of an issue and I commented the link of the pull request.